agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1816,14 +1810,8 @@ bool backhaul_manager::handle_slave_backhaul_message(std::shared_ptr<sRadioInfo>
 
         if (m_eFSMState >= EState::CONNECT_TO_MASTER) {
             LOG(INFO) << "Sending topology notification on reconnected son_slave";
```

Comment:
same as the above comment regarding the log print

Reply:
Same answer.